mandatory/1-fix-functions.js

```diff
@@ -10,30 +10,28 @@
 
 */
 
-function mood() {
-  let isHappy = true;
-
+function mood(isHappy) {
+  //let isHappy = true;
   if (isHappy) {
     return "I am happy";
   } else {
     return "I am not happy";
   }
 }
 
+//let num = 12;
+let isBigEnough = num > 10;
 function greaterThan10(num) {
-  let isBigEnough;
-
-  if (isBigEnough) {
+   if (isBigEnough) {
```

Comment:
Add it here like this: 
```suggestion
   let isBigEnough = num > 10;
   if (isBigEnough) {
```